smoketests/pages/index.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
It's good practice to add that header to the top of every file covered under the MPL v2.

> Q4: I want to use the Mozilla Public License for software that I have written. What do I have to do?
>
> To apply the Mozilla Public License to software that you have written, add the header from Exhibit A of the license to each source code file in your project. Sample headers for various commenting styles are available here. You may also add additional accurate notices of copyright ownership, such as the name of the copyright holder, but this is not necessary.

https://www.mozilla.org/en-US/MPL/2.0/FAQ/